source/adios2/toolkit/remote/XrootdRemote.cpp

```diff
@@ -134,9 +134,8 @@ void myRequest::Alert(XrdSsiRespInfoMsg &aMsg)
 
     // Print what we received
     //
-    //    fprintf(XrdSsiCl::outFile, "%s@%s: Rcvd %d bytes alert: '%s'\n", rName,
-    //    GetEndPoint().c_str(),
-    //            theMsz, theMsg);
+    fprintf(XrdSsiCl::outFile, "%s@%s: Rcvd %d bytes alert: '%s'\n", rName, GetEndPoint().c_str(),
+            theMsz, theMsg);
```

Comment:
seems to be a old debug message (in a log file). IS this still needed?

Reply:
Commented back out.